### DISCUSSION TITLE
Container Registry lifecycle policy

### DISCUSSION THREAD

**Discussion by ohmer** [community/community#25232] [Actions]
<p>Hi there,</p>
<p>Can we set image lifecycle policies on Container Registry?<br>
Can’t find this in the documentation, have I missed something or this is not a feature?</p>
<p>I am trying to make sure we don’t explode our cost while caching Docker layers.<br>
So far I have been using GitHub Actions Cache and it might become too small for our use case.</p>
<p>Without this feature, my best alternative is to use ECR + CodeBuild (keep cache close to compute to avoid performance hit and data transfer hit). Please tell me it exists, I don’t want to write buildspecs :wink:</p>
<p>Cheers,</p>

**Comment by gitstua** [comment]
<p>I don’t know of this feature but you could use this action which I just found (yet to try it but you can review the source): <a href="https://github.com/marketplace/actions/github-package-repository-janitor" class="inline-onebox" rel="noopener nofollow ugc">GitHub Package Repository Janitor · Actions · GitHub Marketplace · GitHub</a></p>

**Comment by gitstua** [comment, marked as answer]
<p>Actually, I just remembered - this is the official one: <a href="https://github.com/marketplace/actions/delete-package-versions" class="inline-onebox" rel="noopener nofollow ugc">Delete Package Versions · Actions · GitHub Marketplace · GitHub</a></p>
<p>Does that help?</p>

**Comment by ohmer** [comment]
<p>Haven’t tried it yet but looks exactly like what I was looking for.<br>
Thanks heaps <a class="mention" href="/gitstua">@gitstua</a>!</p>
